frontend/src/pages/faq/FAQEntryList.tsx

```diff
@@ -17,26 +26,63 @@ import { FAQEntry } from 'src/services/openapi';
  */
 const FAQEntryList = ({ entries }: { entries: Array<FAQEntry> }) => {
   const { t } = useTranslation();
+  const { enqueueSnackbar } = useSnackbar();
+
+  const copyUriToClipboard = (
+    event: React.MouseEvent<HTMLElement>,
+    anchor: string
+  ) => {
+    navigator.clipboard.writeText(window.location.toString() + `#${anchor}`);
+    enqueueSnackbar(t('faqEntryList.questionURLCopied'), {
+      variant: 'success',
+      autoHideDuration: SNACKBAR_DYNAMIC_FEEDBACK_MS,
+    });
+  };
+
   return (
     <>
       {entries.map((entry) => {
         const answerParagraphs = entry.answer.split('\n');
         return (
           <Box key={`q_${entry.name}`} mb={4}>
-            <Typography id={entry.name} variant="h4" gutterBottom>
-              {entry.question}
-            </Typography>
+            <Paper square>
+              <Box p={2} pb="1px">
+                <Grid container>
+                  <Grid item xs={11}>
+                    <Typography id={entry.name} variant="h4" gutterBottom>
+                      {entry.question}
+                    </Typography>
+                  </Grid>
+                  <Grid item xs={1}>
+                    <Box
+                      display="flex"
+                      alignItems="center"
```

Comment:
![Screenshot 2022-10-16 at 20 30 00](https://user-images.githubusercontent.com/8818081/196051929-ae11b066-80c9-4b1b-8c17-01b05bb28c0d.png)

The button and question title are not well aligned. Somehow the height of the button forced higher height on the title's Grid container. This also means more blank space before the answer's paragraphs.

(not a high priority to fix and not blocking for merging the PR)

Reply:
It's true, this is due to the button padding. I disabled it in my first draft but I realized removing the padding makes the clickable area of button much more smaller.

I'll try to add a `padding-top` on the title.

Reply:
Fixed by https://github.com/tournesol-app/tournesol/pull/1165/commits/a8b731aa52b7c47f801ae80d85c982fd6c76f02f.